src/app/switcher/switcher.service.ts

```diff
@@ -31,6 +31,7 @@ export interface Program {
 export interface ProgramConfig {
   theme_color?: string;
   card_style?: string;
+  review_rating?: boolean;
```

Comment:
Just a suggestion. I think it would be more clear to call it `has_review_rating` or something similar to indicate that it is a boolean.